### PR TITLE
Fomo #3623

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/fomo/fomo.app.mjs [hunk 1]
```diff
@@ -1,11 +1,32 @@
+import { axios } from "@pipedream/platform";
```
[the remark this answers is not in the document]
Comment (lcaresia) [reply components/fomo/fomo.app.mjs:1]: In the docs they recommend to use `api` lib, so I prefered to use the `axios`.

image

File: components/fomo/fomo.app.mjs [hunk 2]
```diff
@@ -22,6 +22,13 @@ export default {
         ...args,
       });
     },
+    async createEvent(args = {}) {
```
[the remark this answers is not in the document]
Comment (lcaresia) [reply components/fomo/fomo.app.mjs:25]: I prefer to stay all methods in app file, because in the future this can be an NPM lib.

**Review by michelle0927 — APPROVED** [review]

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**